### DISCUSSION TITLE
vercel deployment

### DISCUSSION THREAD

**Discussion by PapaYawAsiamah** [vercel/community#2408] [Help]
### Summary

I'm trying to deploy my astro app to vercel but I'm getting this error message

### Additional information

Invoke Error 	{"errorType":"Error","errorMessage":"aborted","code":"ECONNRESET","stack":["Error: aborted","    at connResetException (node:internal/errors:717:14)","    at Socket.socketCloseListener (node:_http_client:462:19)","    at Socket.emit (node:events:525:35)","    at TCP.<anonymous> (node:net:322:12)","    at TCP.callbackTrampoline (node:internal/async_hooks:130:17)"]}
[](url)

**Comment by amyegan** [comment]
Hi @PapaYawAsiamah! Based on the error info provided, I wonder if there is a problem with environment variables or connection to an external API. One quick thing to check is that your project's [Environment Variables](https://vercel.com/docs/concepts/projects/environment-variables#) are set for the appropriate environments and that the values are correct (i.e. tokens are not expired, no typos).

If that doesn't lead you to a solution, please share more error information and context. If you have a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share, that would also be very helpful for debugging 🙏
